core/src/main/java/bio/terra/pearl/core/service/notification/TriggerService.java

```diff
@@ -121,4 +140,15 @@ public void applyDiff(StudyEnvironmentChange change, StudyEnvironment destEnv, P
             PublishingUtils.applyChangesToVersionedConfig(configChange, this, emailTemplateService, destEnv.getEnvironmentName(), destPortalEnv.getPortalId());
         }
     }
+
+    @Override
+    public List<String> getAdditionalPublishIgnoreProps() {
+        return List.of("emailTemplateId", "emailTemplate", "versionedEntity");
+    }
+
+    @Override
+    /** we have to re-implement this method since this class implements both interfaces, and otherwise the default methods collide */
```

Comment:
Just for my own understanding - what exactly does it mean when something is both study and portal publishable?

Reply:
it means it's attached at both the study and portal level.  For triggers, triggers can either be attached to studies, or attached to portals.  We don't currently use the ability to have portal-level triggers, but the thinking was we'd need it for e.g. alerts on registration or other account-level events